src/coreclr/jit/emitjmps.h

```diff
@@ -46,6 +46,12 @@ JMP_SMALL(lt    , ge    , blt    )  // LT
 JMP_SMALL(gt    , le    , bgt    )  // GT
 JMP_SMALL(le    , gt    , ble    )  // LE
 
+#elif defined(TARGET_LOONGARCH64)
+
+JMP_SMALL(jmp   , jmp   , b      )
```

Comment:
Do you plan to add at some point definitions for other branch instructions?
If so, you can add `TODO comment` saying this.